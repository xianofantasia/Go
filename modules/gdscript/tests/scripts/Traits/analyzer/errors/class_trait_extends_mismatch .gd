trait SomeTrait extends Node:
	pass

class SomeClass extends Object:
	uses SomeTrait

func test() -> void:
	print("ok")
