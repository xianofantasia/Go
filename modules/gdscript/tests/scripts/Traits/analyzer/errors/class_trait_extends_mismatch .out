GDTEST_ANALYZER_ERROR
>> ERROR at line 5: "SomeTrait" trait extends "Node" does not match class inheritance.
